Library/Formula/gdal.rb

```diff
@@ -308,3 +323,4 @@ def caveats
     system "#{bin}/ogrinfo", "--formats"
   end
 end
+
```

Comment:
In the Ruby Style guide https://github.com/styleguide/ruby referenced from https://github.com/Homebrew/homebrew/tree/master/share/doc/homebrew#readme

End each file with a blank newline.